INFO     root:conftest.py:208 === Starting test_negative_create_invoice_invalid_product Test (Negative) ===
INFO     root:conftest.py:209 API Response: 404 | {"ErrorCode":"10004","ErrorDescription":null,"ErrorDetail":"Product with Id: 100 not found.","Url":"http://api.dev.nerdunited.net/api/invoices","type":null,"title":"A problem occurred processing your request","status":404,"detail":"Nerd.Common.Exceptions.EntityNotFoundException\u00601[Nerd.Shopping.Core.Entities.Product]: Product with Id: 100 not found.\n   at Nerd.Shopping.Core.Services.ProductService.GetCurrentPriceAsync(Int32 productId, CancellationToken cancellationToken) in /src/Nerd.Shopping/Nerd.Shopping.Core/Services/ProductService.cs:line 115\n   at Nerd.Shopping.Core.Services.InvoiceService.BuildLineItems(CreateInvoiceRequest request, List\u00601 exchangeRates, CancellationToken cancellationToken) in /src/Nerd.Shopping/Nerd.Shopping.Core/Services/InvoiceService.cs:line 201\n   at Nerd.Shopping.Core.Services.InvoiceService.CreateInvoiceAsync(Int32 memberId, CreateInvoiceRequest request, CancellationToken cancellationToken) in /src/Nerd.Shopping/Nerd.Shopping.Core/Services/InvoiceService.cs:line 107\n   at Nerd.Shopping.Api.Controllers.V1.InvoicesController.Post(CreateInvoiceRequest request, CancellationToken cancellationToken) in /src/Nerd.Shopping/Nerd.Shopping.Api/Controllers/V1/InvoicesController.cs:line 51\n   at lambda_method2766(Closure , Object )\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.\u003CInvokeActionMethodAsync\u003Eg__Awaited|12_0(ControllerActionInvoker invoker, ValueTask\u00601 actionResultValueTask)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.\u003CInvokeNextActionFilterAsync\u003Eg__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State\u0026 next, Scope\u0026 scope, Object\u0026 state, Boolean\u0026 isCompleted)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.\u003CInvokeInnerFilterAsync\u003Eg__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.\u003CInvokeFilterPipelineAsync\u003Eg__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.\u003CInvokeAsync\u003Eg__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.\u003CInvokeAsync\u003Eg__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.\u003CInvoke\u003Eg__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\n   at Nerd.Common.Logging.NerdApiGlobalLogScopeMiddleware.InvokeAsync(HttpContext httpContext) in /src/Nerd.Common/Logging/NerdApiGlobalLogScopeMiddleware.cs:line 41\n   at Microsoft.AspNetCore.Authorization.Policy.AuthorizationMiddlewareResultHandler.HandleAsync(RequestDelegate next, HttpContext context, AuthorizationPolicy policy, PolicyAuthorizationResult authorizeResult)\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\n   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.\u003CInvoke\u003Eg__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)","instance":null}
INFO     root:conftest.py:219 Test 'test_negative_create_invoice_invalid_product' (Negative) SUCCESS.
INFO     root:conftest.py:220 === Ending test_negative_create_invoice_invalid_product Test (Negative) ===